wpimath/src/main/java/edu/wpi/first/math/kinematics/SwerveDriveKinematics.java

```diff
@@ -233,7 +233,11 @@ public Twist2d toTwist2d(SwerveModulePosition... wheelDeltas) {
    */
   public static void desaturateWheelSpeeds(
       SwerveModuleState[] moduleStates, double attainableMaxSpeedMetersPerSecond) {
-    double realMaxSpeed = Collections.max(Arrays.asList(moduleStates)).speedMetersPerSecond;
+    Double[] absoluteModuleSpeeds = new Double[moduleStates.length];
```

Comment:
```suggestion
    var absoluteModuleSpeeds = new double[moduleStates.length];
```

Reply:
I can't use `double` because `Collections.max` won't work, and the whole thing will crash if I do.

Reply:
At least use `var`. Specifying the type twice is redundant.

Reply:
In that case, at least use `var`. Specifying the type twice is redundant.

Reply:
So would `var absoluteModuleSpeeds = new Double[moduleStates.length];` work in this case?

Reply:
Yes.